internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml

```diff
@@ -33,11 +33,8 @@ DeploymentConfiguration:
       {{- end }}
     {{- end }}
     Enable: true
-  {{- else }}
-    AlarmNames: []
-    Enable: false
-  {{- end}}
```

Comment:
We still want to keep this because otherwise removing ABR setting from the manifest won't result in CFN updating on ECS service to disable ARB. In order to turn it off, we have to explicitly specify this in CFN template.

Reply:
> Removing the ABR config from the manifest should just remove all the Alarms keys from the CFN

Yes this is correct.

> Shouldn't that be sufficient to disable the feature?

I don't think so...ECS update service api is stateful so you have to explicitly disable the alarm otherwise it won't pick up any changes. This is due to the desired count issue ECS service used to have. The benefit is for updateService API when you don't specify desiredCount, ECS just assumes you don't want to update fields that are not specified. Same logic applies to the alarm field.

Reply:
What a thorny, annoying trade-off.